app/views/flood_risk_engine/enrollments/steps/_local_authority_postcode.html.erb

```diff
@@ -1,26 +1,9 @@
-<fieldset>
-  <legend class="visuallyhidden"><%= t('.legend') %></legend>
-
-  <%= form_group_and_validation(form, :postcode) do -%>
-    <%= f.label :postcode,  t('.label') , class: 'form-label' %>
-    <span class="form-hint"><%= t('.hint') %></span>
-    <%= f.text_field 'postcode', class: 'form-control form-postcode form-control-char-10' %>
-  <% end %>
-
-  <% if(form.manual_entry_enabled?) %>
-    <p>
-      <%= link_to( t('flood_risk_engine.enrollments.addresses.manual_entry'),
-                   new_address_path( enrollment: form.enrollment,
-                                     addressable: form.enrollment.organisation,
-                                     postcode: form.postcode,
-                                     address_type: :primary,
-                                     step_back_to_postcode: false,
-                                     target_step: form.target_step
-                   ),
-                   class: 'change-postcode-button'
-          )
-      %>
-    </p>
-  <% end  %>
-
-</fieldset>
+<%=
+  render(
+    "postcode",
+    f: f,
+    form: form,
+    addressable: form.enrollment.organisation,
+    address_type: :primary
+  )
+%>
```

Comment:
Looks like you and Tom are on the same page DRYing up addresses with partials - see his PR #131 
May need to coordinate with him depending on who merges first